frontend/src/components/reports/ReportingSearchCard.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-form ref="form" v-model="isFormComplete" class="ma-2">
+    <v-card class="pa-6">
+      <v-row no-gutters>
+        <v-col cols="12" lg="6" class="mb-6 mb-lg-0">
+          <!-- <v-row>
+            <v-col cols="12" sm="6" lg="2" xl="3" class="pb-0">
+              <AppLabel>Report Type(s):</AppLabel>
+            </v-col>
+            <v-col cols="12" sm="9" lg="8" xl="7">
+              <v-select
+                v-model.lazy="selectedReportTypes"
+                :items="reportTemplates"
+                :disabled="loading"
+                :rules="rules.required"
+                item-title="surveyTemplateName"
+                variant="outlined"
+                label="Select Report Type(s)"
+                chips
+                multiple
+                return-object>
+                <template #prepend-item>
+                  <v-list-item title="Select All" @click="toggleAllReportTypes">
+                    <template #prepend>
+                      <v-checkbox-btn
+                        :color="someReportTypesSelected ? '#003366' : undefined"
+                        :indeterminate="someReportTypesSelected && !allReportTypesSelected"
+                        :model-value="someReportTypesSelected" />
+                    </template>
+                  </v-list-item>
+                  <v-divider class="mt-2" />
+                </template>
+              </v-select>
+            </v-col>
+          </v-row> -->
+          <v-row>
+            <v-col cols="12" sm="6" lg="2" xl="3" class="pb-0">
+              <AppLabel>Status(s):</AppLabel>
```

Comment:
Status(s) looks a little odd since the plural of Status is Statuses.

Reply:
Agreed! I copied how we do it in other screens. it means the user can select 1 or multiple statuses. Let me know if I should update it to Status(es)
![image](https://github.com/bcgov/ecc-ofm/assets/119005543/387b8a55-bbe5-4611-a718-b2eff987c47e)

Reply:
Status(es) would be more correct so let's update.

Reply:
it's updated.